content/en/docs/setup/platform-setup/ucloud/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: UCloud
+description: Instructions to setup an UCloud Kubernetes cluster for Istio.
+weight: 70
+skip_seealso: true
+aliases:
+    - /docs/setup/kubernetes/prepare/platform-setup/ucloud/
+    - /docs/setup/kubernetes/platform-setup/ucloud/
+keywords: [platform-setup,uk8s,ucloud]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+Follow these instructions to prepare an
+[UCloud Container Service for Kubernetes(UK8S)](https://docs.ucloud.cn/uk8s/README)
+cluster for Istio.
+You can deploy a Kubernetes cluster to UCloud quickly and easily, which fully supports Istio.
+
+## Procedure
+
+1. Log on to the [UCloud UK8S console](https://console.ucloud.cn/uk8s/manage), and click **Create Cluster** button.
+
+1. Choose dedicated or managed cluster version. If dedicated cluster is chosen, 3 master nodes are required. If you choose managed version, the masters and core components like api-server, etcd, scheduler will be maintained by UCloud.
```

Comment:
Is there anything different between `dedicated` or `managed` when trying to install istio?

Reply:
No，the  dedicated  version means user should manage the master node by themself. The managed cluster version means UCloud will manage the master node. But user has admin role in both cluster version. So there is no difference between dedicated and managed version when trying to install istio.

Reply:
I think it would be better if you can add some screenshots?

Reply:
+1

Reply:
Screenshots has been added for deploy uk8s cluster.